webapp/packages/product-default/src/UpdateInstruction.tsx

```diff
@@ -9,7 +9,8 @@ import { Code } from '@cloudbeaver/core-blocks';
 import type { InstructionComponent } from '@cloudbeaver/core-version-update';
 
 export const UpdateInstruction: InstructionComponent = function UpdateInstruction({ version, containerId, className }) {
-  const id = containerId || 'cloudbeaver';
+  const id = `containerId("${containerId || 'cloudbeaver'}")`;
+
```

Comment:
seems weird in the case when containerId is presented and correct (because now the script itself is invalid)
I propose to replace `id` with constant `{your container id}`